README.md

```diff
@@ -84,6 +84,12 @@ Optional settings:
 
 Ariadne Codegen implements a plugin system that enables further customization and fine-tuning of generated Python code. It’s documentation is available separately in the [PLUGINS.md](https://github.com/mirumee/ariadne-codegen/blob/main/PLUGINS.md) file.
 
+### Built-in plugins
+
+There are some built-in plugins that ships with `ariadne_codegen`:
+
+* [`ariadne_codegen.ShorterResultsPlugin`](plugins/shorter_results.py) - Unwrap
```

Comment:
Added your suggested quote for now!

Reply:
> Sorry, I saw your comment about that but didn't know if you meant in this project or a separate one. I'll move this inside the current project!

Same project. I just want to keep official plugins separate from rest of codebase under specific namespace, like how Ariadne or Django does it.

> I also wanted to avoid over documenting it in the main README,

Don't worry about that. We can always change it later when we become unhappy with it :D